src/_guides/language/language-tour.md

```diff
@@ -144,21 +144,21 @@ mind:
     from executing at all; a run-time error results in an
     [exception](#exceptions) being raised while the code executes.
 
-<ul>
-  <li id="runtime-modes" class="no_toc" markdown="1">
-  Dart has two <em id="runtime-modes">runtime modes</em>:
-  production and checked. We recommend that
-  you develop and debug in checked mode, and deploy to production mode.
-
-  *Production mode* is the default runtime mode of a Dart program,
-  optimized for speed. Production mode ignores [assert
-  statements](#assert) and static types.
-
-  *Checked mode* is a developer-friendly mode that helps you catch some
-  type errors during runtime. For example, if you assign a non-number to a
-  variable declared as a `num`, then checked mode throws an exception.
-  </li>
-</ul>
+-   Dart has two <em>runtime modes</em>:
+    production and checked. We recommend that
```

Comment:
Also, the link to /dart-2.0 above is misformatted (a [ should be a ], so the link doesn't work.).